test/junit/scala/tools/nsc/settings/TargetTest.scala

```diff
@@ -13,23 +13,30 @@
 package scala.tools.nsc
 package settings
 
-import org.junit.{Assert, Test}
+import org.junit.{Assert, Test}, Assert.{assertEquals, assertFalse, assertTrue, fail}
```

Comment:
What happened to the proposal for

```scala
import org.junit.{Assert.{assertEquals,assertFalse,assertTrue,fail}, Test}
```
?

Reply:
Hello Rust!